### PR TITLE
Create navbar

### PR DESCRIPTION
**Pull request by leefreemanxyz**
~~This code will not currently work. We have paths that we need to link to which do not currently exist as we haven't received the tickets to do so yet.~~ The image can be replaced with a better asset when one is available/provided.

This should work and is tested.

screen shot 2017-03-28 at 18 02 11

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+</head>
```
Comment (Woulei) [inline app/views/shared/_navbar.html.erb]: Don't think this should be in the partial?

Comment (leefreemanxyz) [inline app/views/shared/_navbar.html.erb]: Updated in commit cc391fb

**Comment by miriamtocino** [comment]
I don't understand why you cannot add the proper links to your navbar, since they should redirect to the pages/routes that you created this morning 😄

**Comment by leefreemanxyz** [comment]
The correct links are in now, and the class on the buttons should correctly toggle.

**Comment by miriamtocino** [comment]
:shipit: !!!

**Pull request merged**